packages/paste-design-tokens/tokens/themes/dark/global/border-color.yml

```diff
@@ -13,107 +13,95 @@ props:
     value: "{!palette-gray-60}"
     comment: Default border color
   color-border-weak:
-    value: "{!palette-gray-70}"
+    value: "{!palette-gray-80}"
     comment: Weak border color
   color-border-weaker:
-    value: "{!palette-gray-80}"
+    value: "{!palette-gray-90}"
     comment: Weaker border color
-  color-border-inverse-weaker:
-    value: "{!palette-gray-0-transparent-20}"
-    comment: Weaker border on inverse backgrounds. Must be used on color-background-body-inverse.
-  color-border-inverse:
-    value: "{!palette-gray-0-transparent-50}"
-    comment: Border on inverse backgrounds. Must be used on color-background-body-inverse.
-  color-border-inverse-strong:
-    value: "{!palette-gray-0-transparent-70}"
-    comment: Strong border on inverse backgrounds. Must be used on color-background-body-inverse.
-  color-border-inverse-stronger:
-    value: "{!palette-gray-0-transparent-90}"
-    comment: Stronger border on inverse backgrounds. Must be used on color-background-body-inverse.
-  color-border-inverse-strongest:
-    value: "{!palette-gray-0}"
-    comment: Strongest border on inverse backgrounds. Must be used on color-background-body-inverse.
```

Comment:
Just making double sure, these now inherit from the default theme, correct?

Reply:
Yeah! Inverse border tokens are now the same in Default and Dark theme